src/components/BookStore.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useState } from 'react';
```

Comment:
- [ ] Kindly make sure to not disable Linters because the role of Linters is to assist us to write clean code so it doesn't look professional to disable it in your repo while it helps us to achieve quality. Kindly fix it.